Code/Tests/BlurTest/Makefile

```diff
@@ -19,7 +19,7 @@ LIBS=\
 	$(PSPDEV)/psp/sdk/lib/libpspsdk.a\
 	$(PSPDEV)/psp/sdk/lib/libpsppower.a\
 	$(PSPDEV)/psp/sdk/lib/libpsprtc.a\
-	$(PSPDEV)/psp/sdk/lib/libpspgum.a\
+	$(PSPDEV)/psp/sdk/lib/libpspgum_vfpu.a\
```

Comment:
this _seems_ unnecessary? regular gum works fine here..

Reply:
currently sceGUM is broken in newer pspsdk revisions.

Reply:
Well, the code builds and runs as-is. 🤷 
(fwiw I've only tested with the hardware I have available - phat 150 and slim 3.90 m33-3)